flutter_secure_storage_web/lib/flutter_secure_storage_web.dart

```diff
@@ -130,26 +127,24 @@ class FlutterSecureStorageWeb extends FlutterSecureStoragePlatform {
     required Map<String, String> options,
   }) async {
     final iv =
-        html.window.crypto!.getRandomValues(Uint8List(12)).buffer.asUint8List();
+    (web.window.crypto.getRandomValues(Uint8List(12).toJS) as js_interop.JSUint8Array).toDart.buffer.asUint8List();
```

Comment:
Since `getRandomValues` updates its arguments, the following implementation would reduce the `toDart` and `toJS` conversions.

```dart
final iv = Uint8List(12);
web.window.crypto.getRandomValues(iv.toJS);
```

https://github.com/google/webcrypto.dart/blob/232f5d6faeeefe232726f1a2938df695ad16406b/test/crypto_subtle_test.dart#L64